Makefile

```diff
@@ -178,3 +178,18 @@ build-webhook:
 .PHONY: docker-build-webhook
 docker-build-webhook:
 	docker build -f cmd/admission-webhook-server/Dockerfile -t $(ADMISSION_WEBHOOK_SERVER) .
+
+.PHONY: check-update-assets
```

Comment:
Those checks should run `git status --porcelain` check before `make update-assets`, otherwise you get false positive.